src/lib/holocene/table/selectable-table.svelte

```diff
@@ -0,0 +1,75 @@
+<script lang="ts">
+  import { omit } from '$lib/utilities/omit';
+  import type { SvelteComponent } from 'svelte';
+  import TableHeaderRow from './table-header-row.svelte';
+  import TableRow from './table-row.svelte';
+  import Table from './table.svelte';
+
+  type T = $$Generic;
+  type Item = T & { id: string };
+
+  export let allSelected: boolean = false;
+  export let items: Item[];
+  export let TableRowChildren: typeof SvelteComponent;
```

Comment:
Would have liked this to have typed props matching the generic `Item` type, but `SvelteComponentTyped<Item, {}, {}>` was not working here.